src/ssz/type/container.zig

```diff
@@ -285,6 +294,21 @@ pub fn VariableContainerType(comptime ST: type) type {
             break :blk out;
         };
 
+        pub fn equals(a: *const Type, b: *const Type) bool {
+            inline for (fields) |field| {
+                if (comptime isFixedType(field.type)) {
```

Comment:
if there's no difference between the if and else branch, then you can remove the branching entirely